packages/server/lib/modes/run.js

```diff
@@ -1519,6 +1519,14 @@ module.exports = {
               if (!options.quiet) {
                 renderSummaryTable(runUrl)(runSpecs)
               }
+
+              const hasFailedSpec = runSpecs.runs.find((run) => run.failures > 0)
+
+              if (hasFailedSpec) {
+                process.exit(-1)
```

Comment:
I have always seen `process.exit(1)` be the unix standard for exiting with an error.
```suggestion
                process.exit(1)
```